utils/frame/benchmarking-cli/src/overhead/weights.hbs

```diff
@@ -37,17 +37,17 @@ parameter_types! {
 	{{#if (eq short_name "block")}}
 	/// Time to execute an empty block.
 	{{else}}
-	/// Time to execute a NO-OP extrinsic eg. `System::remark`.
+	/// Time to execute a NO-OP extrinsic, for example `System::remark`.
 	{{/if}}
 	/// Calculated by multiplying the *{{params.weight.weight_metric}}* with `{{params.weight.weight_mul}}` and adding `{{params.weight.weight_add}}`.
 	///
-	/// Stats [ns]:
+	/// Stats [NS]:
```

Comment:
Iiuc this is nanoseconds, correct? If so, we should add this as a whitelist item in the `spellcheck.conf` rather than adjusting the case.

Reply:
Yep. Having some of these whitelisted would be nice :pray:

Reply:
Adding something like https://github.com/paritytech/polkadot/blob/master/scripts/gitlab/spellcheck.toml with https://github.com/paritytech/polkadot/blob/master/scripts/gitlab/lingua.dict